### PR TITLE
add commerce url to store details

### PR DESCRIPTION
**Pull request by icaraps**
Together with https://github.com/hlxsites/sony-biotech/commit/e5b1efc3a90f545a2902d8c9d896bc10f173d858 to allow cart updates on plp

### CODE REVIEW

**Comment by icaraps** [comment]
cc @arumsey

File: src/context/store.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ interface StoreProps extends WithChildrenProps {
   context?: QueryContextInput;
   apiUrl: string;
   apiKey: string;
+  commerceUrl: string;
```
Comment (arumsey) [inline src/context/store.tsx:29]: The store context already has an `apiUrl`. Are both needed? Currently the `apiUrl` is hardcoded in the webpack config but that could be changed to allow for an override via the store config if needed.

The hardcoded value is currently: `https://catalog-service.adobe.io/graphql`

Comment (icaraps) [reply src/context/store.tsx:29]: I was wondering too if we could reuse `apiUrl` but I was under the impression you don't get the same results using `https://catalog-service.adobe.io` vs the commerceUrl cc @herzog31

Comment (herzog31) [reply src/context/store.tsx:29]: Transactional operations don't use catalog service, but the core GraphQL endpoint. This value is stored under the `commerce-core-endpoint` key in configs.json.

Comment (icaraps) [reply src/context/store.tsx:29]: Ok so we need commerce and catalog endpoints 👍

Comment (arumsey) [reply src/context/store.tsx:29]: The current `apiUrl` in the PLP storefront could have been named better since it is only meant for live search requests.

**Pull request merged**